storage/migrations/5_windowpost.go

```diff
@@ -0,0 +1,24 @@
+package migrations
```

Comment:
It'll be 8 now. This is going to continue to be a problem while we have an integer index for migrations. Perhaps it would be better to use timestamps to make the index collisions (at least between migrations touching mutually exclusive schema) less painful to reconcile. But maybe this is a short term problem while dev velocity is high. Just a showerthought.

Reply:
The migrations package we use expects a monotonic counter. It will fail if there are two migrations for the same schema version. I'll add a test for that so we see it at build time instead of migration time.